docs/roomHandlers/fishingRoom.md

```diff
@@ -0,0 +1,41 @@
+# Fishing Room
+
+A room with a beach and a boat dock. The player can fish from dock or leave. Fishing may result in catching a fish.
+
+## States
+
+- **AtEntrance**: The player is on a dock with a fishing pole
```

Comment:
Should this keep track of how many fish you have caught?

Reply:
Updated.